### PR TITLE
Config/initial Initial files added

### PR DESCRIPTION
**Pull request by gergana95**
Added readme, gitignore, html and css files.

### CODE REVIEW

File: normalize.css [hunk 1]
```diff
@@ -0,0 +1,427 @@
+/*! normalize.css v6.0.0 | MIT License | github.com/necolas/normalize.css */
```
Comment (superKalo) [inline normalize.css]: That's not the latest version of [Normalize.css](http://necolas.github.io/normalize.css/). Probably, you'd want to use v7.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8"/>
+	<title>Surfing</title>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="index.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1"/>
+</head>
```
Comment (superKalo) [inline index.html]: Please check out the [recommended minimum](https://github.com/joshbuchea/HEAD#recommended-minimum) for the tags present in the `<head>` section.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
```
Comment (superKalo) [inline index.html]: You can add `lang="en"` attribute to the `<html>` tag too.

File: normalize.css (hunk 1 above)
Comment (gergana95) [reply normalize.css]: Updated normalize file!

File: index.html (hunk 2 above)
Comment (gergana95) [reply index.html]: Added meta compatible ie tag and changed places. Meta tags are above title tag.

File: index.html (hunk 3 above)
Comment (gergana95) [reply index.html]: Added lan="en" to html tag.

File: index.html (hunk 2 above)
Comment (superKalo) [reply index.html]: okay!

**Comment by superKalo** [comment]
Okay, great! 👍 

You can now merge your first PR 💣 🎊 🚀

**Pull request merged**